app/services/process_submission_service.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# rubocop:disable all
```

Comment:
I've crossed a threshold of loc and complexity.
This class needs a refactor, don't want to do it as a part of this feature.